README.md

```diff
@@ -8,9 +8,27 @@ It is built using [Tailwind CSS](https://tailwindcss.com/) and [Eleventy](https:
 It is hosted on GitHub Pages and every commit to the `main` branch is automatically
 deployed using GitHub Actions.
 
+## Prerequisites 
+
+### Linux/MacOS
+* `git` and `gitbash`
```

Comment:
```suggestion
* `git`
```

No such thing as gitbash on a proper os :stuck_out_tongue_winking_eye:

Reply:
lol, copy+paste error (gitbash is a toy for windows ;))